aten/src/ATen/native/Convolution.cpp

```diff
@@ -566,7 +580,7 @@ at::Tensor _convolution(
   auto k = weight.ndimension();
   c10::IntArrayRef weight_sizes = weight.sizes();
   int64_t dim = k - 2;
-  
+
```

Comment:
Empty space.

Reply:
Thanks will fix.

Reply:
Same here.  Removed trailing space on save.  Will disable this setting on the editor, but let's let this pass.  It feels wrong adding that trailing space back.